index.js

```diff
@@ -161,7 +161,7 @@ function CompassCompiler(inputTree, files, options) {
   this.options = merge(true, this.defaultOptions);
   merge(this.options, options);
   options = this.options;
-  options.files = (options.files instanceof Array) ? options.files : [];
+  options.files = (options.files && options.files.length) ? [].concat.apply([], [options.files]) : [];
```

Comment:
You are right :+1:, but sorry to insist, I prefer to change it to a simpler form, it confused me, so it will do with others :)

This line will be enough: `options.files = options.files && [].concat(options.files) || [];`